source/_data/data.json

```diff
@@ -149,6 +149,8 @@
     ]
   },
   "copyright" : "© 2017 Superpedestrian, Inc. All Rights Reserved.",
+  "cartNav" : "https://shop.superpedestrian.com/cart",
```

Comment:
Probably not something related to this work but we might want to figure out how we want to route them in case they have something in a EU store cart since getting redirected to the US store by default would kinda be confusing.

Reply:
this afternoon, I suggested handling that in ecomm, observing the cookie that is set and redirecting within ecomm as appropriate.